test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -332,15 +332,18 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
 
     It "Set-Service can run -Status Stopped to stop a service with dependencies" {
         $servicename = "spooler"
-        $originalStatus = (Get-Service $servicename).Status
-        try {
-            Set-Service -Status Running $servicename
-            $script = { Set-Service -Status Stopped $servicename }
-            { & $script } | Should -Not -Throw
-            (Get-Service $servicename).Status | Should -BeExactly "Stopped"
-        }
-        finally {
-            Set-Service -Status $originalStatus $servicename -ErrorAction SilentlyContinue
+        $service = Get-Service -Name $servicename -ErrorAction SilentlyContinue
```

Comment:
This would report the test as passed not skipped

Reply:
I am sorry, but I don't know how to do it, could you give me any suggestion?